lib/apiservers/engine/backends/system.go

```diff
@@ -93,6 +94,10 @@ func (s *System) SystemInfo() (*types.Info, error) {
 		log.Infof("System.SytemInfo unable to get global status on containers: %s", err.Error())
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), infoTimeout)
```

Comment:
Don't have access to ip address, but we can something like "from Container Management Host".